src/parser/expression.js

```diff
@@ -588,12 +592,22 @@ export default class ExpressionParser extends LValParser {
     node.property = this.parseIdentifier(true);
 
     if (node.property.name !== propertyName) {
-      this.raise(node.property.start, `The only valid meta property for new is ${meta.name}.${propertyName}`);
+      this.raise(node.property.start, `The only valid meta property for ${meta.name} is ${meta.name}.${propertyName}`);
     }
 
     return this.finishNode(node, "MetaProperty");
   }
 
+  parseImportMetaProperty(): N.MetaProperty {
+    const node = this.startNode();
+    const id = this.parseIdentifier(true);
+    this.expect(tt.dot);
+    if (this.options.sourceType !== "module") {
+      this.raise(id.start, "import.meta can only be used in modules");
```

Comment:
Heh, good point. Copied the wording from the "can only be used in functions". Should've taken the wording from import/export statements. Fixed.